pkg/finder/fsfinder.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strings"
 
 	"github.com/Boeing/config-file-validator/pkg/filetype"
+	"golang.org/x/exp/slices"
```

Comment:
You should be able to import slices like a normal standard library import

```suggestion
        "slices"
	"strings"

	"github.com/Boeing/config-file-validator/pkg/filetype"
```

Reply:
Sure. My linter was not allowing this. Sure. Can you trigger the GitHub Action?